mina/caip2.md

```diff
@@ -0,0 +1,115 @@
+---
+namespace-identifier: mina-caip2
+title: Mina Namespace - Chains
+author: Mina Foundation
+discussions-to: https://github.com/ChainAgnostic/namespaces/pull/91
+status: Draft
+type: Standard
+created: 2023-09-15
+requires: CAIP-2
+---
+
+# CAIP-2
+
+*For context, see the [CAIP-2][] specification.*
+
+## Abstract
+
+In CAIP-2 a general blockchain identification scheme is defined. This is the implementation of CAIP-2 for Mina network.
+
+### Mina Namespace
+
+The namespace "mina" refers to the wider Mina ecosystem.
+
+## Rationale
+
+Mina consists of multiple networks: a production network `mainnet`, a testing network `testnet` and networks where new features are trialed. Networks trialing new features are identified by a codename, for example `berkeley`. 
+
+An identifier for a Mina chain consists of the namespace prefix "mina:" followed by human readable reference identifying each network. These identifiers may be extended to include codenames and prefixed to include the type of network. These names are separated by "-". 
+
+The type of network will be:
+- devnet, indicating a stable copy of the network used for developer testing.
+- testnet, indicating a less stable copy of the network where new features can be deployed and tested. 
+
+The names chosen above are taken from the signature schema's for each blockchain.
+
+## Syntax
+
+The blockchain namespace and blockchain identifiers will be in lowercase.
```

Comment:
maximum length? whitespace allowed? other characters, diacritics, etc? a regex never hurts :D

Reply:
I'll add a regex ;D

Reply:
hi @bumblefudge - added a regex and, hopefully ;), clarified what the possible names can be ... does it look ok now?

Reply:
If it is, I'll post a link to think PR in the mina discord so the mina community can comment